examples/MicroBit/src/beacon.adb

```diff
@@ -112,7 +112,7 @@ package body Beacon is
       end if;
 
       --  Set TX packet address
-      Set_Packet (Get_Address (Beacon_Packet));
+      Set_Packet (Memory_Address (Beacon_Packet));
```

Comment:
Ah, I see, you merged it while I was rebasing the whole stuff, that's how this ended up into this pull request.